letsencrypt/tests/cli_test.py

```diff
@@ -175,6 +175,24 @@ def test_certonly_bad_args(self):
         ret, _, _, _ = self._call(['-a', 'bad_auth', 'certonly'])
         self.assertEqual(ret, 'The requested bad_auth plugin does not appear to be installed')
 
+    def test_check_config_sanity_domain(self):
```

Comment:
this test should be moved to `configuration_test.py` (`CheckConfigSanityTest(unittest.TestCase)`)